src/components/BookCard/BookCard.tsx

```diff
@@ -0,0 +1,42 @@
+import { BookData } from '../../models';
+import { getBookImgUrl } from '../../data.utils';
+import { useStateFromQuery } from '../../route.utils';
+import { AppLink } from '../AppLink';
+
+interface BookCardProps {
+  book: BookData;
+  parentUrl?: string;
+}
+
+const BookCard = (props: BookCardProps) => {
```

Comment:
You destructure props and just use `{book, parentUrl}` so that you won't need to use it like `props.book`